release-notes/search-processor.release-notes-2.5.0.md

```diff
@@ -0,0 +1,38 @@
+Compatible with OpenSearch (2.5.0).
+
+### Features
+
+* Add a Docker quickstart script for demo / test ([#25](https://github.com/opensearch-project/search-processor/pull/25))
+* Configure Mend for GitHub.com ([#37](https://github.com/opensearch-project/search-processor/pull/37))
+
+### Enhancements
+
+* Fix codecov after repo renaming ([#41](https://github.com/opensearch-project/search-processor/pull/41))
+* stripped down version of pr stats for only this repository ([#56](https://github.com/opensearch-project/search-processor/pull/56))
+* Update log statements in quickstart helper script ([#79](https://github.com/opensearch-project/search-processor/pull/79))
```

Comment:
Should this be feature?

Reply:
quickstart helper script was implemented since 2.4.0, the #79  is more like an enhancement to the existing feature to update the logging info while running the helper script.

Reply:
'quickstart helper script was implemented since 2.4.0' -> then #25 shouldn't in 2.5 right?
```
* Add a Docker quickstart script for demo / test ([#25](https://github.com/opensearch-project/search-processor/pull/25))
```

Reply:
makes sense, changed https://github.com/opensearch-project/search-processor/pull/79 to labeled with `feature` be at the same category as https://github.com/opensearch-project/search-processor/pull/25